spec/features/user_can_adjust_quantity_of_item_in_cart_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe "User", type: :feature do
+  create_category
+  create_items
+  feature "Cart" do
+    it "can adjust quantity of item" do
+      visit items_path
+      within("#item_#{item_one.id}") do
+        click_button "Add to Cart"
+      end
+      click_link "View Cart"
+      within "#item_row_#{item_one.id}" do
+        fill_in 'Quantity', with: "3"
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.